istio-policy/templates/deployment.yaml

```diff
@@ -95,7 +98,7 @@ spec:
 {{- end }}
 {{- if .Values.global.useMCP }}
     {{- if .Values.global.controlPlaneSecurityEnabled}}
-          - --configStoreURL=mcps://istio-galley.{{ .Values.global.configNamespace }}.svc:15019
+          - --configStoreURL=mcp://localhost:15019
```

Comment:
Is this change consistent with the old installer? I am not clear why this is needed.

Reply:
This is for the policy mixer container to talk to its sidecar, similar to https://github.com/istio/installer/pull/108. In the old installer, Galley doesn't have a sidecar and apparently policy-mixer bypasses its sidecar and talks to Galley directly.